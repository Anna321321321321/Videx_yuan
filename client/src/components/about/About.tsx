import React from 'react';

export default () => (
  <div>
    <b>Terms of Use and Consent Form</b>
    <p>
      Click <a href="/terms">here</a> for the terms of use.
      <br />
      Click <a href="/consent">here</a> for the consent form.
    </p>
    <b>Use of Open Source</b>
    <p>
      Click <a href="/dist/licenses.txt">here</a> for a list of open source used
      in the application.
    </p>
  </div>
);
