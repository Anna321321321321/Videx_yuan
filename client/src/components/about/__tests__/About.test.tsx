import { mount } from 'enzyme';
import React from 'react';
import About from '..';

describe('About Unit Tests', () => {
  it('simple test', () => {
    const wrapper = mount(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
