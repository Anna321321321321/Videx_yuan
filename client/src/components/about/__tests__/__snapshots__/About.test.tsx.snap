// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Unit Tests simple test 1`] = `
<Component>
  <div>
    <b>
      Terms of Use and Consent Form
    </b>
    <p>
      Click 
      <a
        href="/terms"
      >
        here
      </a>
       for the terms of use.
      <br />
      Click 
      <a
        href="/consent"
      >
        here
      </a>
       for the consent form.
    </p>
    <b>
      Use of Open Source
    </b>
    <p>
      Click 
      <a
        href="/dist/licenses.txt"
      >
        here
      </a>
       for a list of open source used in the application.
    </p>
  </div>
</Component>
`;
